tests/unit/FileHelperTest.php

```diff
@@ -7,6 +7,10 @@
 use WP_Mock;
 use WP_Mock\Tools\TestCase;
 
+/**
+ * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
+ */
```

Comment:
Individual processes are needed for all tests that alias classes in Mockery or they interfere with each other.